.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,11 +1,11 @@
-# Rideshare-Rails
-Congratulations! You're submitting your assignment!
+# Rideshare-Rails 
+Congratulations! You're submitting your assignment! 
```

Comment:
Add a note that the two partners should work together on these questions (that was a point of confusion in the past).